__mocks__/electron.ts

```diff
@@ -49,13 +49,15 @@ class TestIpcRenderer {
     }
   }
 
-  send(channel: string, ...args: any[]) {
+  send(channel: string, ...args: any[]) {}
```

Comment:
This function is _actually_ for sending a message to the main process. This is the opposite of how it was being used in the unit tests; the `_simulate` function was added for testing.